flask_wiki/static/js/wiki.js

```diff
@@ -65,3 +65,15 @@ $(document).ready(function () {
     $('.wiki-files .toast').toast('show');
   });
 });
+
+// Change the target modal when an element is clicked
+function updateModal(e) {
```

Comment:
this should be in the document ready. I would change the name to be more specific. I would use jquery onclick instead.